File: BackEnd/Mmo/Mmo_Domain/ModelResponse/OrderUserResponse.cs
namespace Mmo_Domain.ModelResponse;

public class OrderUserResponse
{
    public int OrderId { get; set; }
    public DateTime? OrderDate { get; set; }
    public int ProductId { get; set; }
    public string? ProductName { get; set; }
    public string ShopName { get; set; } = null!;
    public string SellerName { get; set; } = null!;
    public int Quantity { get; set; }
    public decimal TotalPrice { get; set; }
    public string? Status { get; set; }
    public bool hasFeedback { get; set; } = false;
}

File: frontend/src/models/modelResponse/OrderUserResponse.ts
export interface OrderUserResponse {
  orderId: number;
  orderDate: string;
  productId: number;
  productName: string;
  shopName: string;
  sellerName: string;
  quantity: number;
  totalPrice: number;
  status: string;
  payload?: string;
  hasFeedback: boolean;
}
